cmd/snap/cmd_login.go

```diff
@@ -113,6 +113,7 @@ func (x *cmdLogin) Execute(args []string) error {
 
 	email := x.Positional.Email
 	if email == "" {
+		fmt.Fprint(Stdout, i18n.G("Authenticated operation on this device will be under terms of service found at: https://www.ubuntu.com/legal/<some-snap-store-url>")+"\n\n")
```

Comment:
We should move this outside of the `if` -this needs to be presented in the case of their email being given on the CLI too.